subgraph/l2/subgraph.yaml

```diff
@@ -0,0 +1,68 @@
+specVersion: 0.0.4
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: RequestManager
+    source:
+      address: "0xe236d136aD9f6dB125e9ba95eDc1d1A19423Ff0F"
+      abi: RequestManager
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - ClaimMade
+        - ClaimStakeWithdrawn
+        - DepositWithdrawn
+        - FinalityPeriodUpdated
+        - OwnershipTransferred
+        - RequestCreated
+      abis:
+        - name: RequestManager
+          file: ./abis/RequestManager.json
+      eventHandlers:
+        - event: ClaimMade(indexed uint256,uint256,address,uint256,address,uint256,uint256,bytes32)
+          handler: handleClaimMade
+        - event: ClaimStakeWithdrawn(uint256,indexed uint256,address)
+          handler: handleClaimStakeWithdrawn
+        - event: DepositWithdrawn(uint256,address)
+          handler: handleDepositWithdrawn
+        - event: FinalityPeriodUpdated(uint256,uint256)
+          handler: handleFinalityPeriodUpdated
+        - event: OwnershipTransferred(indexed address,indexed address)
+          handler: handleOwnershipTransferred
+        - event: RequestCreated(uint256,uint256,address,address,address,uint256,uint256)
+          handler: handleRequestCreated
+      file: ./src/request-manager.ts
+  - kind: ethereum
+    name: FillManager
+    source:
+      address: "0x85267688CE7677D69d75F288Ff3064431030aF9F"
+      abi: FillManager
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - HashInvalidated
+        - LPAdded
+        - LPRemoved
+        - FillManagerOwnershipTransferred
+        - RequestFilled
+      abis:
+        - name: FillManager
+          file: ./abis/FillManager.json
+      eventHandlers:
+        - event: HashInvalidated(indexed bytes32,indexed bytes32,indexed bytes32)
+          handler: handleHashInvalidated
+        - event: LPAdded(address)
+          handler: handleLPAdded
+        - event: LPRemoved(address)
+          handler: handleLPRemoved
+        - event: FillManagerOwnershipTransferred(indexed address,indexed address)
```

Comment:
Subgraph does this automatically if you have the same event in multiple contracts. It modifies the abi file so that the events are distinguishable from each other and are stored in different entitites.
I would say in the future we can clean up and don't store this event at all because it's not really relevant to us nor the users.

Reply:
Aah, now it makes sense. Thanks for clarifying!